loleaflet/src/layer/tile/CanvasTileLayer.js

```diff
@@ -154,15 +143,14 @@ L.CanvasTilePainter = L.Class.extend({
 
 		var viewBounds = this._map.getPixelBounds();
 		var splitPanesContext = this._layer.getSplitPanesContext();
-		var paneBoundsList = splitPanesContext ?
-		    splitPanesContext.getPxBoundList(viewBounds) :
-		    [viewBounds];
-		var canvasCorePx = new L.Point(this._pixWidth, this._pixHeight);
+		var paneBoundsList = splitPanesContext ? splitPanesContext.getPxBoundList(viewBounds): [viewBounds];
 
-		return { canvasSize: canvasCorePx,
-			 tileSize: tileSize,
-			 viewBounds: viewBounds,
-			 paneBoundsList: paneBoundsList };
+		return {
+			canvasSize: new L.Point(this._pixWidth, this._pixHeight),
```

Comment:
The info that this is core pixels got lost; please either add a comment, or keep the local var.